Resources/Private/Templates/Export/List.html

```diff
@@ -79,7 +79,7 @@ <h4 class="form-section-headline">Included content elements:</h4>
             <f:for each="{files}" as="fileContent" key="fileName" iteration="iterator">
                 <div class="panel panel-space panel-default">
                     <div class="panel-heading" role="tab" id="file-{iterator.cycle}-heading">
-                        <a class="collapsed" data-toggle="collapse" title="{fileName}" data-parent="#file-{iterator.cycle}-content" href="#file-{iterator.cycle}-content" aria-expanded="false" aria-controls="file-{iterator.cycle}-content">
+                        <a class="collapsed" data-bs-toggle="collapse" title="{fileName}" data-bs-parent="#file-{iterator.cycle}-content" href="#file-{iterator.cycle}-content" aria-expanded="false" aria-controls="file-{iterator.cycle}-content">
```

Comment:
Hi @NoelClick,

Thank you for your patch request. Unfortunately this change breaks the toggle in TYPO3 < 11.5
I guess the extension needs to provide both options for compatibility.

Reply:
You are right! I have now additionally added the old attributes.
I have also changed the label and description once again.